doc/install/windows-install.rst

```diff
@@ -13,6 +13,9 @@ Prerequisites
 
 Supported platforms
 -------------------
+.. note::
+
+  Please see the `OS recommendations`_ regarding client support.
```

Comment:
The very next sentence tells users that Windows Server 2019 and Windows Server 2016 are supported, so can I see them skipping the link in the note on the assumption that it would say the same thing.  Perhaps making it say "client packages" instead of just "client" could prevent that?
```suggestion
  Please see the `OS recommendations`_ regarding client packages support.
```